options.default

```diff
@@ -37,10 +37,20 @@ SERVICES_DEFAULT_ADMIN_PASSWORD=adminadmin
 ##
 
 # Kafka
-ENABLE_CONNECT=true
+AETHER_CONNECT_MODE='LOCAL'
```

Comment:
I would not remove the option `ENABLE_CONNECT` it has a different meaning than the `CONNECT_MODE`